packages/web/src/hooks/useMedia.ts

```diff
@@ -1,8 +1,8 @@
 import { useMedia as useMediaQuery } from 'react-use'
 
 export const useMedia = () => {
-  const isDesktop = useMediaQuery('(min-width: 1280px)')
-  const isTablet = useMediaQuery('(max-width: 1280px) and (min-width: 480px)')
+  const isDesktop = useMediaQuery('(min-width: 860px)')
```

Comment:
this is def a big change... down to improve this later, but since we are only using for sign-up, this much more accurately correlates with the ux we want